Signal/src/Profiles/OWSProfileManager.h

```diff
@@ -9,6 +9,8 @@ NS_ASSUME_NONNULL_BEGIN
 extern NSString *const kNSNotificationName_LocalProfileDidChange;
 extern NSString *const kNSNotificationName_OtherUsersProfileDidChange;
 
+extern const NSUInteger kOWSProfileManager_MaxAvatarWidth;
```

Comment:
Just to be difficult, I went with Diameter since `size` usually refers to an instance of CGSize(x, y)